### ISSUE TITLE
Updating the design manual Header 2

### ISSUE THREAD

**Issue by klocraft** [cfpb/design-manual#268]
The current manual Header 2 is Avenir Next 18/20 demi-bold. 

The problem: At current size, Header 2 is too different from Header 1 and too similar to the Subheader and Header 3. As a result, Header 2 is rarely used.

Suggested new size: 24/28, Regular

Suggested new size is based on the Tri-fold template and Leave-behind templates, where it can be used as the main header. The suggested Header 2 has also been used in Handouts.

Suggest continuing to test out this new size then updating the manual accordingly.

**Comment by stephanieosan** [comment]
@klocraft Is this for print or web or both? 

What do we need to do to finalize this decision? Who needs to test this out? Do you have examples of this in action?

**Comment by klocraft** [comment]
@stephanieosan, this change would be for print only. I am gathering instances where designers have used the H2 at this new size and I will continue post examples as I find them. Thus far I have not heard of any problems using the H2 at this size, but am interested in further feedback.

I have found that the suggested change to the H2 allows it to act as the main header on documents where the standard H1 is too big and the standard H2 is too small (see tri-fold brochure template and 4x6 Leave behind template below).

**Trifold title shows an example using the new H2**

![screen shot 2015-02-03 at 6 42 18 pm](https://cloud.githubusercontent.com/assets/8646678/6032261/02134e3a-abd5-11e4-8e9b-64173b5e3b21.png)

**4x6 leave behind template shows an example using the new H2**

![screen shot 2015-02-03 at 6 43 38 pm](https://cloud.githubusercontent.com/assets/8646678/6032276/2d44eb18-abd5-11e4-8b3c-9679df17a85e.png)

**Comment by nataliafitzgerald** [comment]
@klocraft - I know throughout this exploratory phase we have looked at 24 point versus 26 point for Header 2. Can you add visuals here to compare those two options?

**Comment by Dnpizarro** [comment]
@klocraft will work on this for the March Sprint

**Comment by klocraft** [comment]
@nataliafitzgerald, here are some examples of Header 2 at 24 vs 26 pt. I had originally gravitated towards 24pt because of the projects I was using as the tester. However, now that I'm taking a second look, the 26 pt is working pretty well too.

### All headers

![comparison](https://cloud.githubusercontent.com/assets/8646678/6414810/6171b0f6-be68-11e4-8030-aaec51cec29b.png)

### Trifold

![screen shot 2015-02-27 at 10 02 28 am](https://cloud.githubusercontent.com/assets/8646678/6414940/526d901a-be69-11e4-9451-6fac2977e956.png)

### 4x6 Leave Behind

![screen shot 2015-02-27 at 10 05 58 am](https://cloud.githubusercontent.com/assets/8646678/6414956/5ff3f256-be69-11e4-8f52-4e16208d1a4e.png)

### Use case

So far I've found it rare to use the H2 directly after a H1. I've mostly used Header 2 to replace the H1 (when H1 proves too large). Examples include the cover of the trifold brochure and some of the leave behind documents. I've also used the H2 in Handouts that include a Worksheet. And in stand-alone Worksheets.

Long story short. H2 needs to hold its own as the leading header, while remaining small enough to be flexible across different sized print pieces (Especially with our narrower documents). Both sizes accomplish this...although 26 may be winning me over.

I now realize that if we do decide on 24, I should tighten up the leading.

**Comment by Dnpizarro** [comment]
@klocraft I like 26/28pt for the H2, but I'm concerned about the tracking. The 24pt option seems a bit too small for the use cases you're highlighting. Would it be odd if we tried 25/28? I know its not an even number, but does it need to be? Maybe something worth trying...

**Comment by nataliafitzgerald** [comment]
Based on @klocraft and @Dnpizarro's comments it sounds like we may be pushing towards the 26/28. In terms of the tracking, I imagine that with narrow documents we may need to allow for some tightening up of the tracking but we will need to define a parameter for this, something like "no greater than -15." The next step is probably to test this out with real headlines that have come up in the past. The goal would be to see whether long words will fit into this template. I'm thinking of words like "Ombudsman" or "Servicemembers." Are there longer ones we can think of? It looks like "Servicemember" would fit.

**Comment by klocraft** [comment]
Yes, "Servicemeber" does fit, it just often times ends up being the only word on a line, especially for the Tri-fold...that's about as long a word as can fit at 26pt. 

As for tracking. For body copy the general rule is to not go beyond -25 to +25...but even that can become too obvious. We'll have to keep an eye on this.

I can put together some examples focusing on 25 pt vs. 26pt. (And yes @Dnpizarro, it would be odd (literally) if we used 25pt, ha ha, get it?).

**Comment by nataliafitzgerald** [comment]
@klocraft - Approving the new size for Header 2 (26/28). This will need to be updated in all Design templates as well as in the content of the Design Manual. Thanks Kim!

**Comment by klocraft** [comment]
Ahh, a great chance to test out my "how to update" notes. Expect great things!

**Comment by klocraft** [comment]
This issue has been updated in the Design Manual.

**Comment by klocraft** [comment]
Opening this issue back up. I had a dyslexic moment. H2 on the design manual should be 26/28. (Currently it's reading 28/26). @Scotchester, can you help me update.

**Comment by Scotchester** [comment]
Hmm... I thought I caught that and corrected it for you.

**Comment by Scotchester** [comment]
That must've been something else. The CSS for the example also needs to be updated. I'll help you out.

**Comment by klocraft** [comment]
I had thought so too...the design manual is messing with us.
